include/z64item.h

```diff
@@ -2,12 +2,74 @@
 #define Z64ITEM_H
 
 typedef enum {
-    /* 0x00 */ EQUIP_SWORD,
-    /* 0x01 */ EQUIP_SHIELD,
-    /* 0x02 */ EQUIP_TUNIC,
-    /* 0x03 */ EQUIP_BOOTS
+    /* 0 */ EQUIP_SWORD,
+    /* 1 */ EQUIP_SHIELD,
+    /* 2 */ EQUIP_TUNIC,
+    /* 3 */ EQUIP_BOOTS,
+    /* 4 */ EQUIP_MAX
 } EquipmentType;
```

Comment:
```suggestion
    /* 0 */ EQUIP_TYPE_SWORD,
    /* 1 */ EQUIP_TYPE_SHIELD,
    /* 2 */ EQUIP_TYPE_TUNIC,
    /* 3 */ EQUIP_TYPE_BOOTS,
    /* 4 */ EQUIP_TYPE_MAX
} EquipmentType;
```
As an idea, could you add `TYPE` to this enum, which means we wouldn't need the awkward `EQUIP_EQUIP_` enums below. They could just be:
```
typedef enum {
    /* 0 */ EQUIP_SWORD_NONE,
    /* 1 */ EQUIP_SWORD_KOKIRI,
    /* 2 */ EQUIP_SWORD_MASTER,
    /* 3 */ EQUIP_SWORD_BGS,
    /* 4 */ EQUIP_SWORD_MAX
} EquipSword;
```
Or maybe throw in `VALUE` for the actual value of the type, like:
```
typedef enum {
    /* 0 */ EQUIP_VALUE_SWORD_NONE,
    /* 1 */ EQUIP_VALUE_SWORD_KOKIRI,
    /* 2 */ EQUIP_VALUE_SWORD_MASTER,
    /* 3 */ EQUIP_VALUE_SWORD_BGS,
    /* 4 */ EQUIP_VALUE_SWORD_MAX
} EquipValueSword;
```
Basically, the double `EQUIP_EQUIP_` just seems a little awkward to me